### PR TITLE
Add information about the Interop Project to the README

### PR DESCRIPTION
**Pull request by gsnedders**
Split from #390, based on discussion in #393.

### CODE REVIEW

**Review by chrishtr — APPROVED** [review]
This is great, thanks!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
 # The Interop Project
 
-Welcome to the Interop project, an ongoing effort to make the web more interoperable in key areas, prioritized by web developer and user needs. This is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+Welcome to the Interop Project, an ongoing effort to make the web more interoperable in key areas, prioritized by web developer and user needs. This is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform, and worked on by the [Interop Team](./supporters.md).
```
Comment (jgraham) [inline README.md:3]: I would suggest putting "users" before "web developers" everywhere. Ultimately helping developers is only important insofar as it ultimately benefits users. Generally their needs are aligned (although there may be cases e.g. when features have privacy concerns, where they are not), but I don't think we should imply that developers' needs outweigh those of end users.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+
+## The Purpose of the Interop Project
+
+*Improve interoperability significantly for the benefit of web developers and users.*
+
+The goal of the Interop Project is to improve the web by making it easier to make websites and web apps that work in every browser or browser engine at the same time.
+
+This is done by increasing the amount of “interoperability” between browsers — when each browser engine has implemented the same technology the exact same way, as bug-free as possible.
+
+Today’s browsers have made a commitment to implement web technology according to a shared web standard, created in organizations such as the W3C or WHATWG, where technologies such as CSS and HTML are officially defined.
```
Comment (jgraham) [inline README.md:16]: Link to the external organisations.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+
+There is a seemingly infinite amount of work that browser engineering teams could be focused on. The Interop Project provides incentives to focus on the specific and practical work that will help web developers most in the coming year.
```
Comment (jgraham) [inline README.md:18]: I don't think we should frame this as "help web developers most", but maybe something more generic like "have the most positive impact on the web platform"

File: README.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
+
+## Scope
+
+The Interop Project is a collaboration between organizations that implement web technology in browser engines. It’s a shared agreement to prioritize work on certain web technologies to improve interoperability. And to keep track of that work by using automated tests to score how much progress each participating browser has made reaching the shared goals.
```
Comment (jgraham) [inline README.md:22]: It's not an agreement to prioritise work. It's an agreement to create a public metric. Orgs are of course free to prioritise whatever work they like.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,57 @@
...
+### **Requirements for Focus Area proposals**
+
+* This feature has a mature-enough web standard from an established standards development organization such as IETF, Khronos Group, TC39, WHATWG, or the W3C.
+* This feature has high quality tests in WPT or Test262.
+
+### **Guidance for Prioritizing Focus Area proposals**
+
+Beyond the above requirements, in line with the above goal, areas where web developers have had problems using them should be prioritized, whether that is because of differences between browsers, or because they aren’t implemented in all browsers. There are many potential sources for information about such areas, including but not limited to:
+
+* Issues filed about such problems in the Chromium, Gecko, or WebKit bug trackers.
```
Comment (jgraham) [inline README.md:41]: It seems like compat problems observed on real sites should also be mentioned (could be with a link to webcompat.com).

**Review by jgraham — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,5 +1,57 @@
...
+* Ranking as a high-demand feature or as a pain-point consistently in polls and surveys of web developers.
+* Documented examples of sites or libraries working around the fact that a feature is missing or not interoperable.
+
+When assessing the prioritization of existing web technology, it can be helpful to assess how often the technology is being used currently. However, one should also consider if important technology might not be in popular use due to lack of interoperability. Potential sources of data include but are not limited to:
+
+* Browser use counters.
+* Usage recorded via HTTP Archive crawls.
+* Number of mentions on developer resources like Stack Overflow.
+
+There are also guiding values that should be taken into consideration, beyond pure demand:
```
Comment (boazsender) [inline README.md:52]: I think we should remove ", beyond pure demand" because it makes it sound like these priorities are less important than demand.

**Comment by chrishtr** [comment]
Still LGTM with the comments from James and Boaz addressed. I think both of them made some good suggestions.

File: README.md (hunk 3 above)
Comment (foolip) [reply README.md:21]:
```suggestion
There is a seemingly infinite amount of work that browser engineering teams could be focused on. The Interop Project provides incentives to focus on the specific and practical work that will have the most positive impact on the web platform in the coming year.
```

**Review by foolip — APPROVED** [review]

**Review by jgraham — APPROVED** [review]

**Pull request merged**